snapcraft/config.py

```diff
@@ -66,6 +70,18 @@ def is_empty(self):
         return True
 
     def load(self):
+        # Local configurations (per project) are supposed to be static.
+        # That's why it's only checked for 'loading' and never written to.
+        # Essentially, all authentication-related changes, like login/logout
+        # or macaroon-refresh, will not be persisted for the next runs.
+        local_config = '.snapcraft.cfg'
```

Comment:
is it too much to ask to create a `.snapcraft` directory? How much of a burden is an extra file? We are planning on using this layout btw:
```
.plugins/
    snapcraft.yaml
    plugins/
        x_plugin1.py
        x_plugin2.py
        ...
```